admin/testbrew.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python
+# Copyright Hybrid Logic Ltd.  See LICENSE file for details.
+
+"""
+Script executed by Buildbot on the Mac mini build slave, this instantiates the
```

Comment:
This is an operational concern.  It's interesting that this is for OS X and it's interesting that it depends on, say, VMware.  It's not so interesting that Buildbot runs it (maybe we'll switch to Jenkins) or that it runs on a Mac mini (maybe we'll move it to run on a VM or an OS X cloud service).